packages/astro/src/integration/index.ts

```diff
@@ -58,6 +69,9 @@ export const sentryAstro = (options: SentryOptions = {}): AstroIntegration => {
                     telemetry: uploadOptions.telemetry ?? true,
                     sourcemaps: {
                       assets: uploadOptions.assets ?? [getSourcemapsAssetsGlob(config)],
+                      filesToDeleteAfterUpload: uploadOptions?.filesToDeleteAfterUpload
+                        ? uploadOptions?.filesToDeleteAfterUpload
+                        : updatedFilesToDeleteAfterUpload,
```

Comment:
super-l: We could use nullish coalescing here, right? 
```suggestion
                      filesToDeleteAfterUpload: uploadOptions?.filesToDeleteAfterUpload ?? updatedFilesToDeleteAfterUpload,
```